### PR TITLE
Added an error log also change request pipeline

### PR DESCRIPTION
**Pull request by FeloniukM**

### CODE REVIEW

File: backend/Tasque.WebAPI/Middlewares/ExceptionMiddleware.cs [hunk 1]
```diff
@@ -20,11 +23,13 @@ public async Task InvokeAsync(HttpContext httpContext)
             }
             catch (HttpException ex)
             {
+                _logger.LogError("A new http exception has been thrown: {ex.StatusCode}| {ex.Message}", ex.StatusCode, ex.Message);
```
Comment (ihor-il) [inline backend/Tasque.WebAPI/Middlewares/ExceptionMiddleware.cs]: Logs are configured to display message level so you don't have to explain what happened, just pass {status code} - {error message}.

File: backend/Tasque.WebAPI/Middlewares/ExceptionMiddleware.cs [hunk 2]
```diff
@@ -20,11 +23,13 @@ public async Task InvokeAsync(HttpContext httpContext)
...
                 await CreateExceptionAsync(httpContext, ex.StatusCode, new { error = ex.Message });
                 return;
             }
-            catch (Exception)
+            catch (Exception ex)
             {
+                _logger.LogError("Something went wrong: {ex.Message}", ex.Message);
```
Comment (ihor-il) [inline backend/Tasque.WebAPI/Middlewares/ExceptionMiddleware.cs]: Same as above

**Review by ihor-il — CHANGES_REQUESTED** [review]

**Review by Anatolii-Ivanov — DISMISSED** [review]

File: backend/Tasque.WebAPI/Middlewares/ExceptionMiddleware.cs (hunk 1 above)
Comment (FeloniukM) [reply backend/Tasque.WebAPI/Middlewares/ExceptionMiddleware.cs]: Fixed it

**Review by ihor-il — COMMENTED** [review]
Why do you have package-lock.json here?

**Comment by FeloniukM** [comment]
>

**Comment by FeloniukM** [comment]
> Why do you have package-lock.json here?
Because it is not currently available in the develop branch

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by ihor-il — APPROVED** [review]

**Pull request merged**